app/src/components/AddCredentialSlider.tsx

```diff
@@ -21,24 +21,18 @@ const AddCredentialSlider: React.FC = () => {
   const [showGetFoundationCredential, setShowGetFoundationCredential] = useState<boolean>(false)
   const [addCredentialPressed, setAddCredentialPressed] = useState<boolean>(false)
   const [workflowInFlight, setWorkflowInFlight] = useState<boolean>(false)
+  const [workflowConnectionId, setWorkflowConnectionId] = useState<string | undefined>()
+
+  const offers = useCredentialByState(CredentialState.OfferReceived)
+
   const credentials = [
     ...useCredentialByState(CredentialState.CredentialReceived),
     ...useCredentialByState(CredentialState.Done),
   ]
+
   const navigation = useNavigation()
   const [canUseLSBCredential] = useState<boolean>(true)
 
-  useEffect(() => {
```

Comment:
any reason why we're removing this here?

Reply:
Just organizing them all in one place

Reply:
ah okay, I see it now